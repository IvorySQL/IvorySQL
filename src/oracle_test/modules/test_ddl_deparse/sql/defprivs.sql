--
-- ALTER DEFAULT PRIVILEGES
--

ALTER DEFAULT PRIVILEGES IN SCHEMA public
  REVOKE ALL PRIVILEGES ON TABLES FROM public;
