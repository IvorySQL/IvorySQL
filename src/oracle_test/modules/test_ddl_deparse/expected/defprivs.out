--
-- ALTER DEFAULT PRIVILEGES
--
ALTER DEFAULT PRIVILEGES IN SCHEMA public
  REVOKE ALL PRIVILEGES ON TABLES FROM public;
NOTICE:  DDL test: type alter default privileges, tag ALTER DEFAULT PRIVILEGES
